VirtoCommerce.Platform.Web/Infrastructure/Authentication.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Platform.Web.Infrastructure
```

Comment:
Could we avoid of usage the new  Authentication options type, and add LimitedCookiePermissions  to `Authorization` options instead?

Reply:
Of course